app/boards/shields/murphpad/murphpad.zmk.yml

```
@@ -0,0 +1,12 @@
+file_format: "1"
+id: murphpad
+name: MurphPad
+type: shield
+url: https://mechwild.com/product/murphpad/
+requires: [pro_micro]
+exposes: [i2c_oled]
+features:
+    - keys
+    - display
+    - encoder
+    - underglow
```

```suggestion
  - keys
  - display
  - encoder
  - underglow
```
Still was failing the prettier check, so I pulled the repo and ran prettier on it, this fixes metadata validation.